lib/cli/src/commands/create_exe.rs

```diff
@@ -557,35 +548,51 @@ impl CreateExe {
             header_code_path = std::env::current_dir()?;
         }
 
+        println!("Output result to: {}", output_path.display());
+
         /* Compile main function */
         let compilation = {
             let mut include_dir = libwasmer_path.clone();
             include_dir.pop();
             include_dir.push("include");
 
             let mut cmd = Command::new(zig_binary_path);
-            let mut cmd_mut: &mut Command = cmd
-                .arg("cc")
-                .arg("-target")
-                .arg(&zig_triple)
-                .arg(&format!("-L{}", libwasmer_path.display()))
-                .arg(&format!("-l:{}", lib_filename))
-                .arg(&format!("-I{}", include_dir.display()))
-                .arg(&format!("-I{}", header_code_path.display()));
-            if !zig_triple.contains("windows") {
-                cmd_mut = cmd_mut.arg("-lunwind");
+            cmd.arg("build-exe");
+            cmd.arg("-target");
+            cmd.arg(&zig_triple);
+            cmd.arg(&format!("-I{}/", include_dir.display()));
+            cmd.arg(&format!("-I{}/", header_code_path.display()));
+            if zig_triple.contains("windows") {
+                cmd.arg("-lc++");
+            } else {
+                cmd.arg("-lc");
+            }
+            cmd.arg("-lunwind");
+            cmd.arg("-OReleaseSafe");
+            cmd.arg("-fstrip");
+            cmd.arg("-dead_strip");
+            cmd.arg("-dead_strip_dylibs");
+            cmd.arg("-fno-compiler-rt");
+            cmd.arg(&format!("-femit-bin={}", output_path.display()));
+
+            cmd.arg(&object_path);
+            cmd.arg(&c_src_path);
+            cmd.arg(libwasmer_path.join(&lib_filename));
+            if zig_triple.contains("windows") {
+                let mut libwasmer_parent = libwasmer_path.clone();
+                libwasmer_parent.pop();
+                cmd.arg(libwasmer_parent.join("winsdk/ADVAPI32.lib"));
+                cmd.arg(libwasmer_parent.join("winsdk/BCRYPT.lib"));
+                cmd.arg(libwasmer_parent.join("winsdk/KERNEL32.lib"));
+                cmd.arg(libwasmer_parent.join("winsdk/USERENV.lib"));
+                cmd.arg(libwasmer_parent.join("winsdk/WS2_32.lib"));
             }
-            cmd_mut = cmd_mut.arg(&object_path).arg(&c_src_path);
-
             if let Some(volume_obj) = pirita_volume_path.as_ref() {
-                cmd_mut = cmd_mut.arg(volume_obj.clone());
+                cmd.arg(volume_obj.clone());
             }
 
-            cmd_mut
-                .arg("-o")
-                .arg(&output_path)
-                .output()
-                .context("Could not execute `zig`")?
+            println!("{:?}", cmd);
```

Comment:
We can delete this one, too.

Reply:
Can we use `log::debug!()`, then? End users probably don't want to see the full command-line we are using to generate EXEs.

Reply:
yes, I migrated to `log::debug`